manifests/params.pp

```diff
@@ -209,6 +209,8 @@
       } elsif $facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'], '15.04') >= 0 {
         # Ubuntu releases since vivid use systemd
         $service_status = pick($service_status, "/usr/sbin/service ${service_name} status")
+      } elsif $facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'], '18.04') >= 0 {
+        $service_status = pick($service_status, "/usr/sbin/service ${service_name}@*-main status")
```

Comment:
This will never be executed. It must be before the `15.04` block because `18.04` is already greater than `15.04`.

Reply:
Well spotted